storage/innobase/trx/trx0trx.cc

```diff
@@ -725,9 +747,48 @@ dberr_t trx_lists_init_at_db_start()
 		return err;
 	}
 
+  if (srv_operation == SRV_OPERATION_NORMAL) {
```

Comment:
How will this work with `mariadb-backup --prepare --export`?

Reply:
For mariabackup, we always use the scn status in the trx sys page